lib/rubocop/config.rb

```diff
@@ -25,7 +25,7 @@ def initialize(hash = {}, loaded_path = nil)
       @loaded_path = loaded_path
       @for_cop = Hash.new do |h, cop|
         qualified_cop_name = Cop::Registry.qualified_cop_name(cop, loaded_path)
-        cop_options = self[qualified_cop_name] || {}
+        cop_options = self[qualified_cop_name]&.dup || {}
```

Comment:
`&` seems unnecessary because `dup` is always callable.

```suggestion
        cop_options = self[qualified_cop_name].dup || {}
```